src/Vault.sol

```diff
@@ -864,21 +893,30 @@ contract Vault is EIP1967Admin, VaultAccessControl, IERC721Receiver, ICDP, Multi
         delete _vaultNfts[vaultId];
     }
 
-    /// @notice Update stabilisation fee for a given vault (in MUSD weis)
-    /// @param vaultId Id of the vault
-    function _updateVaultStabilisationFee(uint256 vaultId) internal {
-        uint256 deltaGlobalStabilisationFeeD = globalStabilisationFeePerUSDD() -
-            globalStabilisationFeePerUSDVaultSnapshotD[vaultId];
-
-        if (deltaGlobalStabilisationFeeD > 0) {
-            uint256 currentVaultDebt = vaultDebt[vaultId];
-            stabilisationFeeVaultSnapshot[vaultId] += FullMath.mulDiv(
-                currentVaultDebt,
-                deltaGlobalStabilisationFeeD,
-                DENOMINATOR
-            );
-            globalStabilisationFeePerUSDVaultSnapshotD[vaultId] += deltaGlobalStabilisationFeeD;
+    /// @notice Update globalRateFee
+    function _updateRateFee() internal {
```

Comment:
I think it makes sense to return updated value `globalFeesRateDebtAccumulatorD` from this function, to save a bit of gas on storage reads.